src/components/autocomplete/js/autocompleteController.js

```diff
@@ -207,6 +207,17 @@ function MdAutocompleteCtrl ($scope, $element, $mdUtil, $mdConstant, $mdTheming,
     elements.$    = getAngularElements(elements);
   }
 
+	/**
+	 * Finds the correct element that will determine the width
```

Comment:
That's not a valid description. `getWrapTarget` will be not used to only determine the width. As the function says, it's just scanning for the `wrap` target.